### ISSUE TITLE
Last update 5 years ago, is this dead?

### ISSUE THREAD

**Issue by AlexCzar** [asciidoctor/sublimetext-asciidoc#19]

**Comment by hertg** [comment]
Hi @AlexCzar 

I'm wondering the same. As there are no responses to issues / PRs in five years, this repository probably won't come back from the dead. 

I've found a small issue in the syntax highlighting and opened issue #20, but this probably won't ever get implemented in this repo, even if a PR was opened to fix it.

I've seen that you are hosting a fork with more recent updates which is 56 commits ahead of this original repository. Are you trying to maintain a more up-to-date stable copy of the project, or is this mainly just for personal use? I'm asking because I've recently opened [this issue](https://github.com/sharkdp/bat/issues/1290) in the [sharkdp/bat](https://github.com/sharkdp/bat) project where I address that this repository here seems to be dead.

If you are trying to maintain a continuation of this project, would you be comfortable if [sharkdp/bat](https://github.com/sharkdp/bat) changed its dependency for the AsciiDoc highlighting from this original repository to your fork? It's quite a big project with 23k+ stars on Github, so keeping the fork as stable as possible would be a requirement. Unfortunately I'm not comfortable enough to maintain a fork myself, because I don't know anything about this project. I'm just searching for a better alternative for the AsciiDoc syntax highlighting, other than this unmaintained repository.

If you are not comfortable with that idea, I completely understand. 

Maybe there is someone else that might be interested in continuing the project as a fork? @sambacha seems to be hosting a more up-to-date fork of the project too. :)

**Comment by jwillikers** [comment]
@AlexCzar, I'm considering creating a fork and committing to maintaining to it as I use Asciidoctor and Sublime Text everyday. I also enjoy the syntax highlighting `bat` provides. Since this repository belongs to the Asciidoctor organization, I'm going to reach out to them first to see if they can get stuff moving again in this repository.

**Comment by jwillikers** [comment]
Issue created in the main Asciidoctor repository [here](https://github.com/asciidoctor/asciidoctor/issues/3776). If that doesn't get a response, I will try contacting the [Asciidoctor administrators](https://discuss.asciidoctor.org/template/NamlServlet.jtp?macro=app_people&node=1) directly.

**Comment by jwillikers** [comment]
Looks like we just need a volunteer to step forward to continue maintenance of this project.
We can also ask for volunteers on https://discuss.asciidoctor.org or https://gitter.im/asciidoctor/asciidoctor.

@AlexCzar and @sambacha, are either of you interested in volunteering to maintain this project?

**Comment by sambacha** [comment]
I have no problem contributing, I keep all my text-editor stuff normally in
this repo: https://github.com/sambacha/authoring

To be honest, it would be much easier to generate an ANTLR4 grammar file
for ASCIIDOC then convert that into the sublime syntax, that way it will
not involve alot of hand coding, which is what the current state of affairs
is for sublime syntax highlighting seems to be for asciidoc.

**Comment by jwillikers** [comment]
@sambacha Yeah, I noticed that handwritten code and it does look difficult to maintain. Generating it from a grammar would make a lot of sense. It sounds like you'd rather be in a contributor role rather than maintainer, though. Is that right?
